CHANGELOG.md

```diff
@@ -15,3 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## 1.0.0 - 2018-10-30
 
 - First major release
+
+## 1.0.1 - 2018-11-16
+
+- Added namespaces to classes and added autoloading
```

Comment:
Hi Richard. Sorry, I should have provided more guidance with this change. As this change hasn't been committed into master yet, it should be listed under the _unreleased_ section. We also need to specify what type of change this is and I usually like referring back to the pull request for reference.

You can find more about the format that is expected by checking https://keepachangelog.com/en/1.0.0/

It should be something like

```
### Added
- Added namespaces to all classes (PR #19)

### Changed
- Removed lti prefixes from function names as no longer required (PR #19)
```

Reply:
fixed